WooCommerce/Classes/ViewRelated/Orders/Order Creation/EditableOrderViewModel.swift

```diff
@@ -199,6 +208,22 @@ final class EditableOrderViewModel: ObservableObject {
     ///
     @Published var selectedProductViewModel: ProductInOrderViewModel? = nil
 
+    /// Keeps track of selected/unselected Products, if any
+    ///
+    @Published var selectedProducts: [Product?] = []
+
+    /// Keeps track of selected/unselected Product Variations, if any
+    ///
+    @Published var selectedProductVariations: [ProductVariation?] = []
```

Comment:
The future refactor may remove these altogether... however, there's no need to use optionals within the array. If a Product(Variation) is nil, just don't add it to the array. It doesn't make sense to say "we have currently selected products: 'nil, product 1, and nil'" – we would just say "we have currently selected products: 'product 1'"

Reply:
Definitely, that makes sense. Removed on https://github.com/woocommerce/woocommerce-ios/pull/9203/commits/417f5c5dcd8e60316cfa32523580242514e07d35